packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,551 @@
+{
+	"enable_central_scene": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Central Scene",
+		"defaultValue": 1
+	},
+	"multilevel_enable": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"valueSize": 4,
+		"defaultValue": 1
+	},
+	"multilevel_upper_function": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Upper Switch Value",
+		"description": "Allowable range: 0-99",
+		"valueSize": 4,
+		"defaultValue": 255
+	},
+	"multilevel_lower_function": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Lower Switch Value",
+		"valueSize": 4,
+		"defaultValue": 0
+	},
+	"multilevel_duration_function": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration",
```

Comment:
In the manual for zba7140, the unit for this template changes based on the value. 

0 : Use dimmer default timer value. (Default)
1 – 127 : Duration in seconds.
128 – 255 : Duration in minutes (minus 127) from 1 – 128 minutes, where 128 is 1 minute.

There is an example of how to deal with this in the style guide. See Parameter Descriptions, Exception 2. If this applies for all files, put the description here, otherwise put it in the relevant file(s).